README.md

```diff
@@ -37,73 +41,128 @@ We have also built a bunch of additional features on top of ODK collect:
+  across their entire suite of applications.
+
+* **Integration with other systems** - Organizations may need to integrate data collection with
+  other systems or workflows, and embedding ODK Collect in their own application allows them to do
+  this seamlessly. By embedding the app, data can be automatically transferred to other systems or
+  databases, reducing the need for manual data entry and minimizing errors.
+
+* **User experience** - Embedding ODK Collect in an existing application can improve the user
+  experience, as users don't have to switch between different apps to collect and manage data. This
+  can result in increased productivity and efficiency.
+
+* **Access control and security** - Embedding ODK Collect in their own application allows
+  organizations to enforce strict access controls and security measures to protect sensitive data.
+  They can also store data on their own servers or in their own cloud storage, giving them more
+  control over how data is managed and secured.
+
+* **Workflow management** - By embedding ODK Collect in their own application, organizations can
+  more easily manage and track data collection workflows. They can assign tasks to specific users or
+  teams, monitor progress, and generate reports and analytics.
 
 ## Git
 
 We closely follow the stable releases of ODK Collect and build extensions on top of them.
 
 ![Git Versioning](./GitFlowOdkCollectExtension.png)
 
-***collect-stable-release:*** This branch tracks the stable releases of the original getodk/collect
-repository. This branch is updated only when a new stable release is made available in the original
-repository. No active development is done in this branch.
+* **collect-stable-release**  - This branch tracks the stable releases of the original
+  getodk/collect repository. This branch is updated only when a new stable release is made available
+  in the original repository. No active development is done in this branch.
 
-***develop:*** This branch is used for active development. All feature branches are merged into this
-branch, and it is updated regularly. This branch is intended to be used for experimental and ongoing
-development work.
+* **develop** - This branch is used for active development. All feature branches are merged into
+  this branch, and it is updated regularly. This branch is intended to be used for experimental and
+  ongoing development work.
 
-***feature/$FEATURE_NAME:*** These branches are used for active development work, with each branch
-focusing on a specific feature or functionality. They are created from the develop branch, and when
-work is completed, they are merged back into develop.
+* **feature/$FEATURE_NAME** - These branches are used for active development work, with each branch
+  focusing on a specific feature or functionality. They are created from the develop branch, and
+  when work is completed, they are merged back into develop.
```

Comment:
We need to document a way to enable contributors to set up a test ODK server and Firebase instance.

Reply:
Good idea. Let me check if ODK has a documented way to do this already.